lib/server/ServerRequest.js

```diff
@@ -3,11 +3,13 @@
 var ServerRequest = {
 
     from: function(expressRequest, environmentConfig) {
+        var rawQuery = (expressRequest.originalUrl && expressRequest.originalUrl.indexOf('?') != -1) ? expressRequest.originalUrl.substring(expressRequest.originalUrl.indexOf('?') + 1) : null;
```

Comment:
As per my discussion with Wayne, we decided against using qs. This was to avoid loading the module and building the query string from the query object, when that could be simply achieved by using the substring operation.
Will change the code to add a function for line 6 and avoid long code line.